src/integrationTest/java/uk/gov/hmcts/reform/sscs/service/IssueFurtherEvidenceServiceIt.java

```diff
@@ -447,30 +448,40 @@ public void appealWithAppellantAndMultipleOtherPartiesAndMultipleFurtherEvidence
         assertEquals(3, documentCaptor.getAllValues().get(3).size());
         assertEquals(3, documentCaptor.getAllValues().get(4).size());
```

Comment:
These and some of the changes you made below can be covered by:
```
        Assertions.assertThat(documentCaptor.getAllValues())
            .hasSize(5)
            .extracting(ArrayList::new)
            .allSatisfy(pdfList -> Assertions.assertThat(pdfList)
                .hasSize(3)
                .extracting(Pdf::getName)
                .contains("evidence-document2", "evidence-document")
            )
```

Reply:
we still need to check the list contain  "evidence-document" and "evidence-document2". Use my coding can make sure the pair of the pdfDocumentRequest and documentCaptor consist and no change the original logic

Reply:
I know this is good suggest, but we are aiming to dismiss the blocking of the ordering check, not restructuring the checking

Reply:
I actually prefer Benson's approach in this particular case.

Surely the test code will be shorter with Lucas' approach but I learned over time that (counter-intuitively) there's value  on occasion in the tests' code being somewhat repetitive. In this particular case I had to think a lot less to understand Benson's code.

I'll make a judgment call here and say we'll keep this test as written by Benson.

I'll mark this conversation as resolved.